src/index.js

```diff
@@ -1,15 +1,45 @@
-import fs from 'fs-extra';
+import fs from 'fs';
 import Plugin from 'broccoli-plugin';
 import md5Hex from 'md5-hex';
 import path from 'path';
 import { default as _logger } from 'heimdalljs-logger';
 import heimdall from 'heimdalljs';
+import FSTree from 'fs-tree-diff';
+import { Promise } from 'rsvp';
 
 // rollup requires this, so old version of node need it
 import 'es6-map/implement';
 
 const logger = _logger('broccoli-rollup');
 
+class Entry {
```

Comment:
@hjdivad / @krisselden we should consider extracting `entry` as its own module, so we can share them between the various projects.